### PR TITLE
Data 3240 return error on Elastic exception

### PR DESCRIPTION
**Pull request by liatiusim**

### CODE REVIEW

File: persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java [hunk 1]
```diff
@@ -1947,8 +1954,11 @@ public T catchingExecuteInClassLoader(boolean logError, Object... args) {
                 if (logError) {
                     logger.error("Error while executing in class loader", t);
                 }
+                if (throwExceptions) {
+                    throw new RuntimeException(t);
```
Comment (shirbr) [inline persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java:1958]: log?

File: wab/src/main/java/org/apache/unomi/web/EventsCollectorServlet.java [hunk 2]
```diff
@@ -62,12 +62,20 @@ public void destroy() {
...
+        }
     }
 
     @Override
     protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
-        doEvent(req, resp);
+        try {
+            doEvent(req, resp);
+        } catch (Throwable t) { // Here in order to return generic message instead of the whole stack trace in case of not caught exception
+            resp.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, "Internal server error");
```
Comment (shirbr) [inline wab/src/main/java/org/apache/unomi/web/EventsCollectorServlet.java:77]: logs?

Comment (liatiusim) [reply wab/src/main/java/org/apache/unomi/web/EventsCollectorServlet.java:79]: Done

File: persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java (hunk 1 above)
Comment (liatiusim) [reply persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java:1958]: From some reason they have a boolean for that (logError on lines 1954-1955)

**Pull request merged**